src/js/utils/getDatesDiff.js

```diff
@@ -1,4 +1,8 @@
-import {DateTime} from 'https://moment.github.io/luxon/es6/luxon.js';
+// import {DateTime} from 'https://moment.github.io/luxon/es6/luxon.js';
+const {
+  DateTime
+} = require("luxon");
```

Comment:
давай лучше везде единоомбразно делать, на ES-модулях
```suggestion
import {
  DateTime
} from "luxon";
```